### ISSUE TITLE
What is the api used at "url = "http://192.168.126.171:5001/api"?

### ISSUE THREAD

**Issue by zxr445116086** [ThisIsSoMe/CCKS2019-CKBQA#7]
seems like this api does word split, part-of-speech tagging, and dependency parsing.
Dir: CCKS2019-CKBQA/Question_classification/data/convert_data.py

**Comment by ThisIsSoMe** [comment]
Well, it is a dependency parsing api. I tried to use dependency parsing to make a difference, but failed. Actrually you can just ignore bert-bilstm-word model and DO NOT SET use_syntax. This will be fine.
For now, i suggest you just ignore this python file.

**Comment by zxr445116086** [comment]
> Well, it is a dependency parsing api. I tried to use dependency parsing to make a difference, but failed. Actrually you can just ignore bert-bilstm-word model and DO NOT SET use_syntax. This will be fine.
> For now, i suggest you just ignore this python file.

If I ignore this file and run 'python main.py', there will be this error message "FileNotFoundError: [Errno 2] No such file or directory: 'train_data/train_syntax_word.tsv'". 
If I ignore bert-bilstm-word model, how can I do question classification?

**Comment by zxr445116086** [comment]
maybe you can simplify main() function in "Question_classification/BERT_LSTM_word/main.py", it contains too much unused code for a simple classification model.

**Comment by ThisIsSoMe** [comment]
> maybe you can simplify main() function in "Question_classification/BERT_LSTM_word/main.py", it contains too much unused code for a simple classification model.

yes, you are right. I update the detail of readme. And the simplification is my new goal.
